.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- If this is your first pull request: sign the CLA with this GitHub app: https://cla-assistant.io/renovatebot/renovate -->
+<!-- Make sure the `Allow edits and access to secrets by maintainers` checkbox is checked on this pull request. -->
+
+## Changes:
+
+<!-- Describe what behavior is changed by this PR. -->
+
+## Context:
+
+<!-- Describe why you're making these changes if it's not already explained in a corresponding issue. -->
+<!-- If you're closing an existing issue with this pull request, use the keyword Closes #issue_number -->
+
+## Documentation (please check one with an [x])
+
+- [ ] I have updated the documentation, or
+- [ ] No documentation update is required
+
+## How I've tested my work (please tick one)
+
+I have verified these changes via:
+
+- [ ] Code inspection only, or
+- [ ] Newly added unit tests, or
+- [ ] No new tests but ran on a real repository, or
+- [ ] Both unit tests + ran on a real repository
+
+<!-- Do you have any suggestions about this PR template? Edit it here: https://github.com/renovatebot/renovatebot.github.io/edit/build/.github/pull_request_template.md -->
+
+<!-- Please do not force push to this PR's branch after you have created this PR, as doing so makes it harder for us to review your work. -->
+<!-- PRs will always be squashed by us when we merge your work. Commit as many times as you need in this branch. -->
```

Comment:
Is the repository already set-up so that you can only squash-merge?

Reply:
Yes

Reply:
Okay I can resolve this then. 😄